olive/systems/azureml/aml_system.py

```diff
@@ -89,29 +89,35 @@ def __init__(
         is_dev: bool = False,
         accelerators: List[str] = None,
         olive_managed_env: bool = False,
-        requirements_file: Union[Path, str] = None,
```

Comment:
Should we update python env system with corresponding system config as well?
https://github.com/microsoft/Olive/blob/main/olive/systems/python_environment/python_environment_system.py